help/destinations/catalog/email-marketing/sendgrid.md

```diff
@@ -0,0 +1,188 @@
+![](../../assets/catalog/email-marketing/sendgrid/03.jpg)
+
+1. Select the **New Account** option and fill in the *Bearer Token* value. This value is the Sendgrid *API Key* we had previously copied in the [prerequisites section](#prerequisites).
+![](../../assets/catalog/email-marketing/sendgrid/04.jpg)
+
+1. Select **Connect to destination** which will check if the Sendgrid *API Key* we provided is valid and accordingly display a *Connected* status. If the **Connected** status is shown as green we are also shown the next [step](#parameters) to fill in additional information fields.
+![](../../assets/catalog/email-marketing/sendgrid/05.jpg)
+
+### Connection parameters {#parameters}
+
+While [setting up](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/connect-destination.html?lang=en) this destination, you must provide the following information:
+
+*  **[!UICONTROL Name]**: The name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: An optional description that will help you identify this destination in the future.
+![](../../assets/catalog/email-marketing/sendgrid/06.jpg)
+
+## Activate segments to this destination {#activate}
+
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination. 
+
+Refer to the below images for details specific to this destination.
+
+* Select a segment that you setup from the list of available segments
+![](../../assets/catalog/email-marketing/sendgrid/11.png)
+
+* After selecting **Add new mapping** you are shown the mapping page to map the source fields to the Sendgrid API target fields.
+![](../../assets/catalog/email-marketing/sendgrid/13.png)
+
+![](../../assets/catalog/email-marketing/sendgrid/14.jpg)
+
+> [!IMPORTANT]
+> The **Source field** attribute *"Email"* should be mapped to the **Target field** attribute *"external_id"* as shown in the subsequent image.
+![](../../assets/catalog/email-marketing/sendgrid/15.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/16.jpg)
+
+* Similarly the remaining [!DNL Adobe Experience Platform] fields and attributes are mapped to the Sendgrid attributes.
+![](../../assets/catalog/email-marketing/sendgrid/17.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/18.jpg)
+
+* A preview of some of the mappings is as below
+![](../../assets/catalog/email-marketing/sendgrid/19.jpg)
+
+* The comprehensive list of supported mappings that can be set up for the [Sendgrid Marketing Contacts API](https://api.sendgrid.com/v3/marketing/contacts) are listed below.
+
+| Name | Field | Type | Description | Limits |
```

Comment:
Unfortunately, with the way the columns are labeled, to me the table is still more confusing than helpful. It is still not clear to me if these are XDM fields that we are describing or target attributes? Based on the screenshot below, I assume that these are target attributes (attributes that Sendgrid accepts and can interpret), but there is no clear indication, so I just don't know. 

I still think that the table is a good idea, but it needs some tweaks. My suggestions:

1. combine name and field columns in a single column and rename to be clear: Sendgrid attribute. This is assuming that I have a correct understanding of what we are presenting here.  
2. in the table description just above the table, be explicit about the fact that the table shows Sendgrid accepted attributes. 
3. Provide a proper Sendgrid documentation link in the table description.